docs/rules/no-unused-styles.md

```diff
@@ -1,6 +1,51 @@
 # Detect unused StyleSheet rules in React components
 When working on a component over a longer period of time, you could end up with unused StyleSheet rules that slipped in over time but are forgotten as you continue to improve your UX/UI design.
 
+
+---
+
+## Options
+
+### `enableImportsCheck: boolean`
+
+`false` - disabled by  default.
+
+> ⚠️ Note: This is an experimental feature and can include bugs
+
+`no-unused-styles` rule works only for single file (because eslint design), 
+and in case when you store styles in the separate files
+this rule doesn't work. So, to enable parsing imported styles you need to enable the next option: 
+
+```js
+// .eslintrc.js
+module.exports = {
+    // ...
+    rules: {
+        'react-native/no-unused-styles': ['error', {enableImportsCheck: true}],
```

Comment:
Hmm I'm inclined to prefer to go for a separate rule as opposed to adding a configuration option for this existing rule. For example a `react-native/no-unused-styles` and `react-native/no-unused-external-styles`. This way, we also do not need files seperated by a `.stable` and `.experimental` suffix.

Reply:
From my point of view it no make sense as resolving local and external using depend on each others